### ISSUE TITLE
Add-on integration with last.fm

### ISSUE THREAD

**Issue by Korb** [kbuffington/foo_enhanced_playcount#9]
> It will also query last.fm and record play times of every scrobble for a song.

https://www.foobar2000.org/components/view/foo_enhanced_playcount

> If a username is supplied, and last.fm support is enabled, it can also retrieve the timestamp of every scrobble of a song as well.

https://wiki.hydrogenaud.io/index.php?title=Foobar2000:Components_1.0/foo_enhanced_playcount#foo_enhanced_playcount_.28Record_time_of_every_play.2C_and_retrieve_Last.fm_scrobbles.29

From the quotes above, it is not clear what exactly the add-on does when interacting with the last.fm site? It is only clear that it does not send information about the tracks being played to the site, because the add-on interface does not have a password field from last.fm. Forum on hydrogenaud.io is not available. Can you rephrase or expand on the feature description?

**Comment by kbuffington** [comment]
Updated and fixed links.
